lms/templates/content_item_selection.html.jinja2

```diff
@@ -15,7 +15,7 @@
     <script async defer src="{{ url }}"></script>
   {% endfor %}
 
-  <script class="js-hypothesis-config" type="text/json">
+  <script class="js-config" type="text/json">
   {
     "authToken": "{{ js_config.authorization_param }}",
```

Comment:
It's a little confusing that this now `class="js-config"` object actually depends on one of the settings from the `js_config` template variable (but ignores the rest). The `js_config` variable's purpose is to be rendered by the base template as a  `class="js-config"` object, which _this_ `class="js-config"` would conflict with :) But I can sort that out in following PRs

Reply:
> But I can sort that out in following PRs

Yes, that was my intention. I forgot to call that out in the description.